### DISCUSSION TITLE
批量导出为CSV文件时,EXCEL打开为乱码

### DISCUSSION THREAD

**Discussion by Jensway** [hiroi-sora/Umi-OCR#564] [General]
感谢作者的工作,自2.1版本后批量OCR生成的CSV文件用EXCEL2021直接打开,OCR内容中文全为乱码,用记事本打开更改为ANSI格式后才正常显示,2.1前的版本不会,如何解决?谢谢!

**Comment by Gavin1937** [comment]
你好，这一般是EXCEL无法正确识别CSV的编码而导致的。[你可以参考这边的解决方法](https://github.com/hiroi-sora/Umi-OCR/discussions/482)。

**Comment by hiroi-sora** [comment]
> 用记事本打开更改为ANSI格式后才正常显示

有点奇怪，v2.1 后csv应该会优先保存为 `ansi` 编码，如果存在无法被ansi兼容的字符，才会向后顺延，保存为 `utf-8` 等编码。

请问你通过OCR得到的 csv ，用记事本打开时是显示什么编码？

**Reply by Jensway** [reply]
> > 用记事本打开更改为ANSI格式后才正常显示
> 
> 有点奇怪，v2.1 后csv应该会优先保存为 编码，如果存在无法被ansi兼容的字符，才会向后顺延，保存为 等编码。`ansi``utf-8`
> 
> 请问你通过OCR得到的 csv ，用记事本打开时是显示什么编码？
感谢回复，导出后的csv文件用记事本打开时的中文显示是正常的，只是用EXCEL打开后显示的是乱码，现在暂时用 [加文1937] 提供的方法来解决，谢谢各位！
